cmd/manager/main.go

```diff
@@ -166,6 +169,9 @@ func parseFlags() {
 		setupLog.Info("Cannot set both --webhooks-only and --no-webhooks")
 		os.Exit(1)
 	}
+
+	// Set hnc namespace
+	webhooks.SetHncNamespace(hncNamespace)
```

Comment:
Move this to `internal/config/namespace.go` instead? It doesn't really belong in "webhooks," the HNC namespace is no more a property of its webhooks than any other part IMO.

Reply:
Thanks for your review! I was relatively new to this project and still lacked some understanding of the structure of the code. And I have moved `SetHncNamespace` to the `internal/config/namespace.go` as you suggested.